guides/doc-Installing_Capsule_Server/master.adoc

```diff
@@ -8,7 +8,7 @@ endif::[]
 
 include::common/attributes.adoc[]
 
-:context: capsule
+:context: {SmartProxy}
```

Comment:
From our meeting: My concern is that if we going to use any other text than "capsule" we also need to change all the "ifeval" statement as they compare only against this word currently. The same for "satellite".

Reply:
Will start testing.

Reply:
Hi @lzap I looked into this and (with @tahliar 's help ) found that we can create an attribute to replace satellite and capsule. I tested and found it to work everywhere:
`ifeval::["{context}" == "{project-context}"]`
`ifeval::["{context}" == "{smart-proxy-context}"]`

`context` is defined as `{project-context}` or `{smart-proxy-context}` in the master.adoc files. 

When we build, the context is set through the attribute. 

I went through both Foreman/Satellite and Capsule/Smart Proxy scenarios and think that this works. 

If you have any thoughts, let me know.